SPECS/containerized-data-importer/cdi.spec

```diff
@@ -16,7 +16,7 @@
 #
 
 Summary:        Container native virtualization
-Name:           containerized-data-importer
+Name:           cdi
```

Comment:
In my opinion, we should also rename the root-folder from `containerzed-data-importer` -> `cdi` :)

Reply:
> In my opinion, we should also rename the root-folder from `containerzed-data-importer` -> `cdi` :)

Done!